### DISCUSSION TITLE
Project Zomboid

### DISCUSSION THREAD

**Discussion by sayuop** [GameServerManagers/LinuxGSM#4549] [General]
Hello, I'm not sure if this is the right place for this kind of question but how do I go about resetting the whole server and starting on a fresh world?

**Comment by dino1816** [comment]
What I do is I go in terminal, locate server world and delete it, 9/10 times it works, havent done it with PZ server so I don't know and if I am not sure I delete entire server folder and redownload it again, copy config files only and then paste them after server fles are redownloaded.

Might be a better way but its how I do it most often

**Comment by gigajoulz** [comment]
I believe the default server name is pzserver, and simply renaming or deleting that folder will generate a new "pzserver" world.
